### ISSUE TITLE
Can Not Change All Fonts to Installed Open Dyslexia Font

### ISSUE THREAD

**Issue by duaneking** [MicrosoftDocs/visualstudio-docs#5383]
I wanted to experiment with an accessibility improving font to help with visibility issues called "open-dyslexic" in an effort to support/help members of my team who have dyslexia cope and be better developers and I'm finding that this documentation is incorrect and somewhat misleading.

I was also interested int the font myself as I noticed that it just generally improved visibility and made text easier to read than the default of Consolas and SageUI.

Sadly the Visual Studio 2019 setting do not seem to work as documented, as I can only find the font available for some of the options, when in reality, I should be able to set any font I want for any of them if visual studio is to be fully inclusive/accessible.

The font is fully installed and I can use it for some font settings in visual studio 2019, but not for all of them.  It just only shows up as an option for some and not others, its really frustrating.

 Font: https://www.dafont.com/open-dyslexic.font

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e2913896-b0f2-5f3c-d733-09fe5c4e86b1
* Version Independent ID: c997c373-c946-0f02-d4c2-0904e6aab42e
* Content: [Change fonts and colors - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-change-fonts-and-colors-in-visual-studio?view=vs-2019)
* Content Source: [docs/ide/how-to-change-fonts-and-colors-in-visual-studio.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/ide/how-to-change-fonts-and-colors-in-visual-studio.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-general**
* GitHub Login: @TerryGLee
* Microsoft Alias: **tglee**

**Comment by WilliamAntonRohm** [comment]
@duaneking -- Duane, thank you for your feedback.  Please consider these resources:​  
- [https://social.msdn.microsoft.com/Forums/vstudio](https://social.msdn.microsoft.com/Forums/vstudio) 
- [https://www.stackoverflow.com](https://www.stackoverflow.com) 
- [https://www.reddit.com/r/VisualStudio/](https://www.reddit.com/r/VisualStudio) 

You are welcome to use the [Report a Problem](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio) tool to report the issue. 

cc @TerryGLee for awareness

**Comment by duaneking** [comment]
I have already tried that and it did not work.

None of these links answer any of this. And honestly, linking me to stackoverflow or reddit when I file a bug is very insulting.  I'm fully aware that issues posted there are often just ignored and I see multiple posts on both sites that have simply been ignored by the visual studio team so I have no reason to think that any posts I make there will be treated differently.

There are many open issues on fonts on both sites that don't seem to be cared about by the Visual Studio team, as the team seems to be ignoring customer accessibility issues posted about this.

This is a pri0 issue for accessibility in my mind.  At least tell us why a font isn't being displayed as an option or call out why accessibility isn't a large priority based on the lack of effort despite the fact these issue keep being reported both here and on the sites you linked to and have not been mitigated or resolved, please.

**Comment by TerryGLee** [comment]
@duaneking We'll take another look at this and report back soon.

**Comment by duaneking** [comment]
Thank You for taking a deeper look, as one of the many who can't because we lack the physical vision due to external factors beyond our control.

I personally lost my eye defending others from being attacked; Not everybody gets to make that choice or sacrifice.  I have known many who are born with disabilities around vision, and I have seen their struggles firsthand.

Anything you can do to improve accessibility for developers with less than 20/20 vision is very important.

**Comment by TerryGLee** [comment]
@duaneking Following up with you... I can repro the issue that you reported; I downloaded the open-dyslexia font and installed it. While I can get that font to work in other Microsoft products (such as Word, Excel, PowerPoint, etc.), I cannot get it to work in Visual Studio.

So, this is indeed product feedback that we need to get to the product team, and the best way to do that is by using the "Suggest a feature" option in the [Visual Studio Developer Community](https://developercommunity.visualstudio.com/). If you are willing to do this, might I ask you to circle back here and add a link to the feature request that you opened? When you do so, I'll happily upvote it. (I thought to open this feature request on your behalf, but you are so articulate in how you state the need for this font that I didn't want to take away the opportunity for you to use your own words for the most good.)

That being said, I took a long look at the [How to: Change fonts and colors in Visual Studio](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-change-fonts-and-colors-in-visual-studio?view=vs-2019) page I did make some changes to it. Of specific interest to you, I think, is a new **[Accessibility options](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-change-fonts-and-colors-in-visual-studio?view=vs-2019#accessibility-options)** section, which details what accessibility feature we offer by way of fonts and colors. It also includes a link to our [Accessibility features of Visual Studio](../ide/reference/accessibility-features-of-visual-studio.md) page that details our accessibility offerings all-up. I think you might notice too that I added a note about how to suggest new features, which will help us improve the product itself.

Overall, I want to thank you for your feedback. It's helped us improve our documentation, which we appreciate. So, on that front, this GitHub Issue did what it's designed to do: improve our documentation. And also, thanks in advance for helping us to improve the product, too; I look forward to seeing (and upvoting) your suggestion in the [Visual Studio Developer Community](https://developercommunity.visualstudio.com/)!

~ Terry
